persistence/src/main/scala/akka/persistence/kafka/journal/JournalAdapter.scala

```diff
@@ -68,7 +68,8 @@ object JournalAdapter {
         val clientId = clientIdOf(config.journal.kafka.producer.common)
         metrics(clientId)
       }
-      KafkaProducerOf[F](producerMetrics)
+      val prefix = clientIdOf(config.journal.kafka.consumer.common)
+      KafkaProducerOf.apply1[F](prefix, producerMetrics)
```

Comment:
there should be a way to disable producer metrics without disabling overall journal metrics

Reply:
why should the metrics be so configurable? I see not much sense in it

Reply:
If they name clash with other things, it's good to be able to disable them.